sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -265,23 +260,23 @@ def _configure_policies(self, **kwargs):
             HttpLoggingPolicy(**kwargs),
         ]
 
-    def _configure_credential(self, credential):
-        # type: (Any) -> None
+    def _configure_credential(
+        self, credential: Optional[Union[AzureNamedKeyCredential, AzureSasCredential, TokenCredential]]
+    ) -> None:
         if hasattr(credential, "get_token"):
-            self._credential_policy = BearerTokenChallengePolicy(  # type: ignore
-                credential, STORAGE_OAUTH_SCOPE
+            self._credential_policy = BearerTokenChallengePolicy(
+                credential, STORAGE_OAUTH_SCOPE # type: ignore
             )
         elif isinstance(credential, SharedKeyCredentialPolicy):
             self._credential_policy = credential  # type: ignore
         elif isinstance(credential, AzureSasCredential):
-            self._credential_policy = AzureSasCredentialPolicy(credential)  # type: ignore
+            self._credential_policy = AzureSasCredentialPolicy(credential) # type: ignore
         elif isinstance(credential, AzureNamedKeyCredential):
-            self._credential_policy = SharedKeyCredentialPolicy(credential)  # type: ignore
+            self._credential_policy = SharedKeyCredentialPolicy(credential) # type: ignore
         elif credential is not None:
             raise TypeError("Unsupported credential: {}".format(credential))
 
-    def _batch_send(self, table_name, *reqs, **kwargs):
-        # type: (str, List[HttpRequest], Any) -> List[Mapping[str, Any]]
+    def _batch_send(self, table_name: str, *reqs: List[HttpRequest], **kwargs) -> List[Mapping[str, Any]]:
```

Comment:
I _think_ this should be:
```suggestion
    def _batch_send(self, table_name: str, *reqs: HttpRequest, **kwargs) -> List[Mapping[str, Any]]:
```

Reply:
Yes, it's not a list of request based on the type hint in core: https://github.com/Azure/azure-sdk-for-python/blob/b83938fe8a238081c0a1fc41cc6f3b07aaed0595/sdk/core/azure-core/azure/core/pipeline/transport/_base.py#L315